scripted_NOW_Example.m

```diff
@@ -53,6 +53,12 @@
 % to set this parameter.
 problem.MaxwellIndex = 100; %In units of (mT/m)^2 ms
 
+% Set the desired orders of motion compensation and corresponding
+% thresholds. Please see TBD for more information on how to set these
+% parameters. maxMagnitude in units s^order / m.
+problem.motionCompensation.order = [1, 2];
+problem.motionCompensation.maxMagnitude = [1e-4, 1e-4];
```

Comment:
Since linear equality is the default the maxMagnitude setting is unused. Add a line saying linear=false?

Reply:
That is a good idea.. alternatively, we can engage the linear equality constraint if the value is 0, else it uses the nonlinear one. This removes the need for specifying method explicitly.

Reply:
Turned into feature request